src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -22,7 +23,8 @@ const AnonymousChannelListScreen = ({route}) => {
     channels: anonChannels,
     goToChatScreen,
     goToPostDetailScreen,
-    goToContactScreen
+    goToContactScreen,
+    goToCommunityScreen
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `goToCommunityScreen` function to the destructuring assignment from `useAnonymousChannelListScreenHook()` is a good improvement. It provides a way to navigate to the community screen, which enhances the app's navigation capabilities.
```diff
+     goToCommunityScreen
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->